postgres-appliance/Dockerfile

```diff
@@ -564,6 +564,6 @@ RUN sed -i "s|/var/lib/postgresql.*|$PGHOME:/bin/bash|" /etc/passwd \
         fi
 
 COPY scripts bootstrap /scripts/
-COPY scm-source.json launch.sh /
+COPY scm-source.json* launch.sh /
```

Comment:
```suggestion
COPY launch.sh /
```

Reply:
I thought we want to keep this as an optional, feature, as it's nice to have commit id info in the image when someone runs the build.sh instead of Dockerfile directly.

Reply:
`registry.opensource.zalan.do/acid/spilo-*` will have scm-source.json anyway, it is injected by CI/CD during the docker push

Reply:
done.